tests/e2e/standalone/init_test.go

```diff
@@ -0,0 +1,254 @@
+		}
+
+		// Ensure a clean environment
+		must(t, cmdUninstall, "failed to uninstall Dapr")
+
+		output, err := cmdInit(daprRuntimeVersion, "--image-registry", "localhost:5000", "--from-dir", "./local-dir")
+		require.Error(t, err, "expected error if both flags are given")
+		require.Contains(t, output, "both --image-registry and --from-dir flags cannot be given at the same time")
+	})
+
+	t.Run("init", func(t *testing.T) {
+		// Ensure a clean environment
+		must(t, cmdUninstall, "failed to uninstall Dapr")
+
+		output, err := cmdInit(daprRuntimeVersion)
+		t.Log(output)
+		require.NoError(t, err, "init failed")
+		assert.Contains(t, output, "Success! Dapr is up and running.")
+
+		homeDir, err := os.UserHomeDir()
+		require.NoError(t, err, "failed to get user home directory")
+
+		daprPath := filepath.Join(homeDir, ".dapr")
+		require.DirExists(t, daprPath, "Directory %s does not exist", daprPath)
+
+		verifyContainers(t, daprRuntimeVersion)
+		verifyBinaries(t, daprPath, daprRuntimeVersion, daprDashboardVersion)
+		verifyConfigs(t, daprPath)
+	})
+}
+
+// verifyContainers ensures that the correct containers are up and running.
+// Note, in case of slim installation, the containers are not installed and
+// this test is automatically skipped.
+func verifyContainers(t *testing.T, daprRuntimeVersion string) {
+	t.Run("verifyContainers", func(t *testing.T) {
+		if isSlimMode() {
+			t.Skip("Skipping container verification because of slim installation")
+		}
+
+		cli, err := dockerClient.NewClientWithOpts(dockerClient.FromEnv)
+		require.NoError(t, err)
+
+		containers, err := cli.ContainerList(context.Background(), types.ContainerListOptions{})
+		require.NoError(t, err)
+
+		daprContainers := map[string]string{
+			"dapr_placement": daprRuntimeVersion,
+			"dapr_zipkin":    "",
+			"dapr_redis":     "",
+		}
+
+		for _, container := range containers {
+			t.Logf("Found container: %v %s %s\n", container.Names, container.Image, container.State)
+			if container.State != "running" {
+				continue
+			}
+			name := strings.TrimPrefix(container.Names[0], "/")
+			if expectedVersion, ok := daprContainers[name]; ok {
+				if expectedVersion != "" {
+					versionIndex := strings.LastIndex(container.Image, ":")
+					if versionIndex == -1 {
+						continue
+					}
+					version := container.Image[versionIndex+1:]
+					if version != expectedVersion {
+						continue
+					}
+				}
+
+				delete(daprContainers, name)
+			}
+		}
+
+		assert.Empty(t, daprContainers, "Missing containers: %v", daprContainers)
+	})
+}
+
+// verifyBinaries ensures that the correct binaries are present in the correct path.
+func verifyBinaries(t *testing.T, daprPath, runtimeVersion, dashboardVersion string) {
+	binPath := filepath.Join(daprPath, "bin")
+	require.DirExists(t, binPath, "Directory %s does not exist", binPath)
+
+	binaries := map[string]string{
+		"daprd":     runtimeVersion,
+		"dashboard": dashboardVersion,
+	}
+
+	if isSlimMode() {
+		binaries["placement"] = ""
```

Comment:
same as runtimeVersion ...

Reply:
`placement --version` does not work, right?